test/end-to-end.js

```diff
@@ -426,6 +426,30 @@ describe('end-to-end', function() {
           done();
         });
     });
+
+    it('hides endpoint GET', function(done) {
+      request(app)
+        .get('/api/Messages')
+        .expect(404, done);
+    });
+
+    it('hides endpoint POST', function(done) {
```

Comment:
I am not sure if it's necessary to test four endpoints. Why exactly four tests, why not only two, or why not one test for each PersistedModel route? I would personally keep the first test only.

However, if you feel it's better to have more tests, then I am ok with that too, just apply the comment above to names.